ecosystem/indexer-grpc/indexer-grpc-cache-worker/src/lib.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::{Deserialize, Serialize};
+use std::{fs::File, io::Read, path::PathBuf};
+
+pub mod constants;
+pub mod worker;
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct IndexerGrpcCacheWorkerConfig {
+    /// Indexer GRPC address.
+    pub indexer_address: String,
+
+    /// Chain ID
+    pub chain_id: u64,
+
+    /// Starting version; if not provided, will start from the latest version in the cache.
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub starting_version: Option<u64>,
```

Comment:
why don't we start from latest version in file store?

Reply:
Going to add in the follow-up PR.